src/main/java/subway/line/Line.java

```diff
@@ -0,0 +1,30 @@
+package subway.line;
+
+public class Line {
+    private Long id;
+    private final String name;
+    private final String color;
+
+    public Line(String name, String color) {
+        this.name = name;
+        this.color = color;
+    }
```

Comment:
`Line(name, color)`는 `Line(id, name, color)`의 subset 으로 보이는데, 
생성자를 delegate 하는게 어떨까요?
```suggestion
    public Line(String name, String color) {
        this(null, name, color);
    }
```

Reply:
네 다른 객체에서도 모두 delegate 하도록 변경하였습니다 !
중복이 많이 줄어든 것 같습니다.